src/components/Editor/index.js

```diff
@@ -443,6 +451,7 @@ class Editor extends PureComponent<Props, State> {
     if (
       line !== null &&
       this.lastJumpLine !== null &&
+      this.lastJumpLine !== line &&
```

Comment:
this was the other half. I'd like to investigate simplifying this logic with a component that is similar to DebugLine